client/src/App.js

```diff
@@ -22,7 +23,13 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route
+          path="/movies/:id"
+          render={props => {
```

Comment:
good job here with render props,  although this step is not necessary until the stretch goals.